### PR TITLE
Add CI setup

### PR DESCRIPTION
**Pull request by IvayloPapazov**
in scope of #10

### CODE REVIEW

**Comment by dincho** [comment]
@IvayloPapazov I've enabled it in CircleCI, could you please do a empty push to trigger it ?

**Review by venimus — APPROVED** [review]

**Comment by tolbrino** [comment]
Could we perform the builds with OTP20 and OTP21 ?

**Comment by venimus** [comment]
> Could we perform the builds with OTP20 and OTP21 ?

May be it should build windows too

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    macos:
+      xcode: "10.0.0"
+
+commands:
+  setup_ubuntu:
+    description: "Setup Ubuntu environment"
+    parameters:
+      OTP_VERSION:
+        type: string
+        default: "20.3.8.23"
```
Comment (venimus) [inline .circleci/config.yml:17]: I'd rather put that as implicit parameter of the 'otp20' job (as `setup_ubuntu` of the otp21)

**Review by venimus — APPROVED** [review]

**Review by dincho — APPROVED** [review]
I hate build matrixes

**Pull request merged**